bin/ghe-restore

```diff
@@ -451,5 +451,5 @@ fi
 echo "Completed restore of $GHE_HOSTNAME from snapshot $GHE_RESTORE_SNAPSHOT"
 
 if ! $cluster; then
-  echo "Visit https://$hostname/setup/settings to review appliance configuration."
+  echo "Visit https://$hostname/setup/settings to review appliance configuration. Settings must be applied to complete any necessary migrations."
```

Comment:
How about:

> You must now visit https://$hostname/setup/settings to review and save appliance configuration

I feel like this is giving a direct command to the user and actually mentions 'saving' the configuration.  Users who are unfamiliar with GHE may not know that hitting 'Save settings' is the equivalent of 'applying settings'?

Reply:
I agree we should probably be a little firmer here, though "you must" feels a little odd to me.

How about:

> Please visit https://$hostname/setup/settings to review and save the appliance configuration.

Reply:
I still feel that sounds almost optional though… How about:

> To complete the restore process, please visit https://$hostname/setup/settings to review and save the appliance configuration.

Then at least we're giving people the reason for doing this.

Reply:
Oooo, nice ✨

Reply:
Hahah words are hard. Hesitated making this change due to the repetition of 'complete' (previous output says it's complete, now I need to do something to complete it). If neither of you think it's an issue I'll go ahead with it :)

Reply:
Good point @keeran. How about changing "Completed" in the line above to "Finished", so users will see:

```
Finished restore of 5.5.5.5:122 from snapshot 20180326T020444
To complete the restore process, please visit https://5.5.5.5/setup/settings to review and save the appliance configuration.
```

Or move to the end of the line:

```
Restore of 5.5.5.5:122 from snapshot 20180326T020444 finished.
To complete the restore process, please visit https://5.5.5.5/setup/settings to review and save the appliance configuration.
```

Reply:
@keeran @lildude good suggestions :thumbsup:

My vote goes for the second one:

```
Restore of 5.5.5.5:122 from snapshot 20180326T020444 finished.
To complete the restore process, please visit https://5.5.5.5/setup/settings to review and save the appliance configuration.
```

Reply:
Thanks chaps! ✨